### DISCUSSION TITLE
How to access and manage custom filters like My Open Issues , Issues assigned to me

### DISCUSSION THREAD

**Discussion by ivishalgandhi** [ankitpokhrel/jira-cli#749] [General] [answered]
How to access and manage custom filters like My Open Issues , Issues assigned to me, other custom filters saved under fav using CLI ?

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @ivishalgandhi, you can do something like this as mentioned in [the FAQ](https://github.com/ankitpokhrel/jira-cli/discussions/283).

```
jira issue list -q"filter='my saved filter'"
```
